Jenkinsfile

```
@@ -106,12 +106,12 @@ def notifyComplete(String buildStatus) {
         echo "Slack notifications are disabled"
         return
     }
-    
+
     // build status of null means successful
     buildStatus =  buildStatus ?: SUCCESS
     def prevBuildResult = currentBuild.getPreviousBuild()?.getResult()
     echo "notifyComplete with status ${buildStatus}, previous build was ${prevBuildResult}"
-        
+
     if (buildStatus == FAILURE)
```

Please change this as per the notes on the issue
```suggestion
    if (buildStatus == FAILURE && prevBuildResult != FAILURE)
```